lib/routes/account.js

```diff
@@ -288,6 +288,20 @@ module.exports = function (
                     )
                 }
               )
+              .then(
+                function (tokens) {
+                  if (request.payload.service === 'sync' && request.payload.reason === 'signin') {
+                    return mailer.sendNewSyncDeviceNotification(emailRecord.email, {})
```

Comment:
We should pass `{ acceptLanguage: request.app.acceptLanguage }` in for the options here, otherwise UUIC the email will always get sent out in English.